### ISSUE TITLE
Consistent button hover style

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/sul-embed#1872]
The Share&Download button hover is an oval. It should be a circle, like the Accessibility button.
<img width="500" src="https://github.com/sul-dlss/sul-embed/assets/2343316/3cba5425-ee2b-4f80-bcc7-6516f0e1e7a1" />

**Comment by jcoyne** [comment]
![Image](https://github.com/sul-dlss/sul-embed/assets/92044/d84eec3b-85db-4c97-8210-b9559d875c6f)

I think the problem here is that the icons themselves have different dimensions.
